python/cuda_cooperative/pyproject.toml

```diff
@@ -1,43 +0,0 @@
-# Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
-#
```

Comment:
question: there's still `pyproject.toml` for `cuda_parallel`, is it a leftover?

Reply:
Fixed